modules/installation-vsphere-infrastructure.adoc

```diff
@@ -29,6 +29,11 @@ You can host the VMware vSphere infrastructure on-premise or on a link:https://c
 |vCenter host   | 7.0 Update 2 or later
 |===
 
+[IMPORTANT]
+====
+You must ensure that the time on your ESXi hosts is synchronized before you install {product-title}. See link:https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vcenterhost.doc/GUID-8756D419-A878-4AE0-9183-C6D5A91A8FB1.html[Edit Time Configuration for a Host] in the VMware documentation.
```

Comment:
Moved note further up, as listing multiple notes does not lead to a good customer UX.